### PR TITLE
Flag draft

### PR DESCRIPTION
**Pull request by talmi**
New draft with the active flag.

### CODE REVIEW

File: drafts/draft-mizrahi-ippm-ioam-flags.xml [hunk 1]
```diff
@@ -330,94 +408,125 @@
...
-	  be possible for active measurement using replicated packets to be 
-	  performed on a subset of the traffic, so as to prevent significant 
-	  overhead.</t>
+      <t>Each of the three flags that are defined in this document may have
+	  performance implications. When using the loopback mechanism a copy of
+	  the data packet is sent back to the sender, thus generating more traffic
+	  than originally sent by the endpoints. Using active measurement with the 
+	  active flag requires the use of synthetic (overhead) traffic. The 
+	  Immediate Export flag triggers exported packets to be sent to an external 
+	  collector, potentially impacting the collector's performance, as well as
```
Comment (barakgafni) [inline drafts/draft-mizrahi-ippm-ioam-flags.xml]: I don't think that this is accurate for all the use cases. For example, when the data is exported to a local collector. Would you agree?

Comment (barakgafni) [reply drafts/draft-mizrahi-ippm-ioam-flags.xml]: the comment applies for the next paragraph as well

File: drafts/draft-mizrahi-ippm-ioam-flags.xml [hunk 2]
```diff
@@ -330,94 +408,125 @@
...
+	  than originally sent by the endpoints. Using active measurement with the 
+	  active flag requires the use of synthetic (overhead) traffic. The 
+	  Immediate Export flag triggers exported packets to be sent to an external 
+	  collector, potentially impacting the collector's performance, as well as 
+	  the paths leading to the collector.</t>
+	  
+	  <t>Each of the three mechanisms has a cost in terms of the network 
+	  bandwidth, and may potentially load the node that analyzes the data.
+	  Therefore, the three mechanisms should be used at a rate that does not 
+	  significantly affect the network bandwidth, and does not overload the
```
Comment (barakgafni) [inline drafts/draft-mizrahi-ippm-ioam-flags.xml]: I would leave this to the operators to consider what they want to do with these tools, I am not sure we should state how much bw should be allocated to what in the network

**Comment by talmi** [comment]
Thanks Barak,

The performance/security-related text was meant to address the comment raised by Mirja (Transport AD) about amplification attacks.
I understand your point that we do not want to mandate a specific mechanism to address these attacks. However, we do need to point out the possible ways to mitigate them.

If you have a specific text edit in mind, it may help the discussion.

**Comment by barakgafni** [comment]
Hi Tal,
Instead of saying that these mechanisms should be limited, can we call for implementations to enable limiting the rate?

**Comment by talmi** [comment]
I suggest the following edits based on Barak's comments:
OLD:
The Immediate Export flag triggers exported packets to be sent to an external collector, potentially impacting the collector's performance, as well as the paths leading to the collector.
NEW:
The Immediate Export flag triggers exported packets to be exported to a collector, which in some cases may impact the collector's performance, or the performance along the paths leading to the collector.

OLD:
Each of the three mechanisms has a cost in terms of the network bandwidth, and may potentially load the node that analyzes the data. Therefore, the three mechanisms should be used at a rate that does not  significantly affect the network bandwidth
NEW:
Each of the three mechanisms has a cost in terms of the network bandwidth, and may potentially load the node that analyzes the data. Therefore, rate limiting may be enabled so as to ensure that the three mechanisms are used at a rate that does not significantly affect the network bandwidth

If these changes are acceptable, I can post the draft before the submission deadline. 

Will appreciate feedback from the authors.

Cheers,
Tal.

**Comment by shwethab** [comment]
The suggested changes looks good to me, if we can publish this draft today we can send it out to ippm. I can do it too if you give a go ahead.
One more update :
John Leddy's email has to be updated to:
john@leddy.net
jleddy@akamai.com

**Comment by talmi** [comment]
Shwetha,
Fine by me.
Please feel free to do that.

**Comment by shwethab** [comment]
> Shwetha,
> Fine by me.
> Please feel free to do that.

Thanks Tal, can you please push the above proposed text update so i can merge this pull request?

**Comment by talmi** [comment]
Done.

**Pull request merged**